inductiva/storage/storage.py

```diff
@@ -0,0 +1,89 @@
+"""Methods to interact with the user storage resources."""
+import inductiva
+from inductiva.client.apis.tags import instance_api
+from inductiva.utils import format_utils
+from typing import Literal
+
+
+def get_space_used():
+    """Returns the occupied storage size in GB."""
+    try:
+        api = instance_api.InstanceApi(inductiva.api.get_client())
+        response = api.get_storage_size()
+        storage_used = round(float(response.body), 3)
+        print(f"Total storage used: {storage_used} GB")
+        return storage_used
+    except inductiva.client.ApiException as api_exception:
+        raise api_exception
+
+
+def listdir(max_results: int = 3,
+            order_by: Literal["size", "creation_time"] = "size",
+            sort_order: Literal["asc", "desc"] = "desc"):
+    """List and display the contents of the user's storage bucket.
+    Args:
+        max_results (int): The maximum number of results to return.
+        order_by (str): The field to sort the contents by.
+        sort_order (str): Whether to sort the contents in ascending or
+        descending order.
+    Returns:
+        list of dict: A list of dictionaries containing information about 
+        the size, the name and the creation time of each content that can
+        easily be converted to a dataframe. 
+
+    This function prints a table with the storage content information: 
+        Name            Size            Creation Time
+        1234            5.68 MiB        29 Sep, 14:12:00
```

Comment:
It should be clear the difference between a file and a directory?
Which of these are files and which of these are directories I can descend into?

Reply:
At this moment, this will have only task directories. Only if the user adds a file directly to the bucket in the GCP platform, it will show single files in the root directories. Files will have an extension, but maybe I can add a / at the end of the folder names.